lib/dataset/test/coords_view_test.cpp

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: BSD-3-Clause
 // Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+#include <gmock/gmock.h>
```

Comment:
Used for...?

Reply:
Forgot to remove. I tried to use a container matcher in the new test but it doesn't like our dict.